### PR TITLE
Add pdf of Intro to scRNA-seq slides for day 2 of training

### PR DESCRIPTION
**Pull request by allyhawkins**
This PR addresses #6 and adds the PDF version of slides for day 2 (day 1 of single-cell RNA-seq) to the `slides` folder. Slides were taken from the [shared drive](https://docs.google.com/presentation/d/1EGijSWUxcfjLpDU9QULYLHxsHMHlI6KPtR1Cs4QWy7A/edit?usp=sharing) found in `Training/Slides/Intro_to_scRNA-seq`.  After converting to pdf, I removed any of the slides that are found in the presentation that are skipped before saving and adding to the folder. Should I also replace the old version of the pdf currently in the google drive slides folder?

### CODE REVIEW

**Comment by jashapiro** [comment]
We will ultimately want these slides to be linked in SCHEDULE.md, and it may be convenient to have that happen as they are uploaded. The caveat is that this would likely create a conflict with #28 at the moment, but I think it may be worth waiting and doing both steps together. Or we could just get this in and then have a separate PR with the schedule links.

Note that these links will be relative links of the form `[slide title](../slides/title.pdf)`

**Comment by allyhawkins** [comment]
> The caveat is that this would likely create a conflict with #28 at the moment, but I think it may be worth waiting and doing both steps together.

If you are okay with it, then I'm okay with waiting until #28 is in and then I'll add the link in with this PR.

**Comment by jaclyn-taroni** [comment]
With https://github.com/AlexsLemonade/2021-june-training/pull/29/commits/ece602bd50a977212cdde0ccd35d1047264781e5, you should be good to add links to these slides in the schedule @allyhawkins

**Review by jashapiro — APPROVED** [review]
LGTM!

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ While we expect to start and end each day on time, the timing of individual modu
 |             | [Consultation session](workshop-structure.md#consultation-sessions) | [Slack channel](../virtual-setup/slack-procedures.md#general-use) |
 | 5:00  PM    | End             |
 | **Day 2**   | **2021-06-29**  <br> [_**Introduction to Single-cell RNA-seq, Day 1**_](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/scRNA-seq/README.md) | 
-| 12:00 PM    | scRNA-seq: Introduction | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+| 12:00 PM    | scRNA-seq: [Introduction (pdf)](../slides/Intro_to_scRNA-seq.pdf) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:26]: Suggesting adding the date to the slide file name (obvs. change the file too!)
```suggestion
| 12:00 PM    | scRNA-seq: [Introduction (pdf)](../slides/2021-06-29_Intro_to_scRNA-seq.pdf) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```

**Pull request merged**